examples/storm-starter/src/jvm/storm/starter/BlobStoreAPIWordCountTopology.java

```diff
@@ -0,0 +1,288 @@
+    private static String key = "key";
+    private static String fileName = "blacklist.txt";
+    private static final Logger LOG = LoggerFactory.getLogger(BlobStoreAPIWordCountTopology.class);
+
+    public static void prepare() {
+        Config conf = new Config();
+        conf.putAll(Utils.readStormConfig());
+        store = Utils.getClientBlobStore(conf);
+    }
+
+    // Spout implementation
+    public static class RandomSentenceSpout extends BaseRichSpout {
+        SpoutOutputCollector _collector;
+
+        @Override
+        public void open(Map conf, TopologyContext context, SpoutOutputCollector collector) {
+            _collector = collector;
+        }
+
+        @Override
+        public void nextTuple() {
+            Utils.sleep(100);
+            _collector.emit(new Values(getRandomSentence()));
+        }
+
+        @Override
+        public void ack(Object id) {
+        }
+
+        @Override
+        public void fail(Object id) {
+        }
+
+        @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("sentence"));
+        }
+
+    }
+
+    // Bolt implementation
+    public static class SplitSentence extends ShellBolt implements IRichBolt {
+
+        public SplitSentence() {
+            super("python", "splitsentence.py");
+        }
+
+        @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("word"));
+        }
+
+        @Override
+        public Map<String, Object> getComponentConfiguration() {
+            return null;
+        }
+    }
+
+    public static class FilterWords extends BaseBasicBolt {
+        String fileName = "blacklist.txt";
+        @Override
+        public void execute(Tuple tuple, BasicOutputCollector collector) {
+            try {
+                String word = tuple.getString(0);
+                Set<String> wordSet = parseFile(fileName);
+                if (!wordSet.contains(word)) {
+                    collector.emit(new Values(word));
+                }
+            } catch (IOException exp) {
+                throw new RuntimeException(exp);
+            }
+        }
+
+        @Override
+        public void declareOutputFields(OutputFieldsDeclarer declarer) {
+            declarer.declare(new Fields("word"));
+        }
+    }
+
+    public void buildAndLaunchWordCountTopology(String[] args) {
+        TopologyBuilder builder = new TopologyBuilder();
+        builder.setSpout("spout", new RandomSentenceSpout(), 5);
+        builder.setBolt("split", new SplitSentence(), 8).shuffleGrouping("spout");
+        builder.setBolt("filter", new FilterWords(), 6).shuffleGrouping("split");
+
+        Config conf = new Config();
+        conf.setDebug(true);
+        try {
+            conf.setNumWorkers(3);
+            StormSubmitter.submitTopologyWithProgressBar(args[0], conf, builder.createTopology());
```

Comment:
I have mentioned how we launch the topology in main, as we want to include topology.blobstore.map config